pyaedt/maxwell.py

```diff
@@ -1951,16 +1951,12 @@ def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
         corresponding to the native AEDT API.  The list of
         keywords here is not exhaustive.
 
-        .. note::
-           This method overrides the ``Analysis.setup()`` method for the HFSS app.
-
         Parameters
         ----------
         setuptype : int, str, optional
-            Type of the setup. Based on the solution type, options are
-            ``"HFSSDrivenAuto"``, ``"HFSSDrivenDefault"``,
-            ``"HFSSEigen"``, ``"HFSSTransient"`` and ``"HFSSSBR"``.
-            The default is ``"HFSSDrivenAuto"``.
+            Type of the setup. Based on the solution type, options are:
+            "Magnetostatic", "EddyCurrent", Transient" and
+            "Electrostatic", "DC Conduction", "Electric Transient", "AC Conduction"
```

Comment:
```suggestion
            ``"AC Conduction"``, ``"DC Conduction"``, ```"EddyCurrent"``, ``"Electric Transient"``,
            ``"Electrostatic"``, ``"Magnetostatic"``, and ``Transient"``.
```
Options should be alphabetized and in double backticks and quotation marks.